### ISSUE TITLE
Metanephrines high in blood

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11016]
**Preferred term label:**
Metanephrines high in blood

**Synonyms**

**Definition (free text, please give PubMed ID)**
Metanephrines are a better test for pheochromocytoma than catecholamines or VMA: https://pubmed.ncbi.nlm.nih.gov/17635948/

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0011976

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Pheochromocytoma
NF1
VHL
MEN2

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
metanephrine and normetanephrine are collectively called fractionated metanephrines
https://pubmed.ncbi.nlm.nih.gov/37300464/
I will define the term thusly

Adding new term: 
- ID: HP:6001470
- label: Abnormal concentration of circulating fractionated metanephrines
- parents: [Abnormal circulating hormone concentration (HP:0003117)](https://hpo.jax.org/app/browse/term/HP:0003117)
- pmid: PMID:37300464
- def: Adrenaline, noradrenaline and dopamine are degraded by catechol-O-methyltransferase (COMT) to form their inactive metabolites metanephrine, normetanephrine and 3-methoxytyramine (3-MT). When there is an indication for biochemical testing, such as suspicion of pheochromocytoma, metanephrine and normetanephrine metabolites are measured in plasma or 24-hour urine. This term represents an elevation of the concentration of metanephrine and normetanephrine in the blood circulation above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11016
- synonyms: Metanephrines high in blood(NONE)
